nvflare/fuel/utils/config_factory.py

```diff
@@ -26,9 +26,9 @@
 class ConfigFactory:
     logger = logging.getLogger(__qualname__)
     _fmt2Loader = {
-        ConfigFormat.JSON: JsonConfigLoader,
-        ConfigFormat.PYHOCON: PyhoconLoader,
-        ConfigFormat.OMEGACONF: OmegaConfLoader,
+        ConfigFormat.JSON: JsonConfigLoader(),
+        ConfigFormat.PYHOCON: PyhoconLoader(),
+        ConfigFormat.OMEGACONF: OmegaConfLoader(),
```

Comment:
It feels weird to import fuel_opt in fuel.
If we want to have OmegaConfLoader and PyhoconLoader, this ConfigFactory should be moved fuel opt?